metrics.json

```diff
@@ -1 +0,0 @@
-jsonnet/telemeter/metrics.jsonnet
```

Comment:
Is this file still needed?

Reply:
Nope

Reply:
Why did you resolve the comment then? :)